foursight_core/react/api/gac.py

```diff
@@ -0,0 +1,62 @@
+import re
+import boto3
+from dcicutils.diff_utils import DiffManager
+from dcicutils.env_utils import short_env_name
+from dcicutils.misc_utils import override_environ
+from dcicutils.obfuscation_utils import obfuscate_dict
+from dcicutils.secrets_utils import get_identity_secrets
+from .misc_utils import sort_dictionary_by_lowercase_keys
+
+
+class Gac:
+
+    @staticmethod
+    def get_secrets_names() -> list:
+        try:
+            boto_secrets_manager = boto3.client('secretsmanager')
+            return [secrets['Name'] for secrets in boto_secrets_manager.list_secrets()['SecretList']]
+        except Exception as e:
+            print("Exception getting secrets")
+            print(e)
```

Comment:
I didn't test this, but I think you want this, assuming you took my setup of `logger` earlier on.
```suggestion
        except Exception as e:
            logger.error(f"Exception getting secrets: {get_error_message(e)}")
```

Reply:
All set.